modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+= Querying a GraphQL API Implementation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+include::reuse::partial$beta-banner.adoc[tag="apikit"]
+
+The following examples showcase the implementation of a GraphQL service to query data using APIkit for GraphQL.
```

Comment:
Not sure about showcase. Suggest illustrate.

Reply:
I like it! Thanks!